src/components/backstage/plugins/Sidebar/NpmDetailsList.js

```diff
@@ -71,31 +71,31 @@ const NpmDetailsList = ({ npmData, npmDataLoadingState }) => {
         foregroundColor="#ecebeb"
       >
         <rect x="0" y="0" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="0" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="0" rx="3" ry="3" width="100" height="24" />
         <rect x="0" y="40" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="40" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="40" rx="3" ry="3" width="100" height="24" />
         <rect x="0" y="80" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="80" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="80" rx="3" ry="3" width="100" height="24" />
         <rect x="0" y="120" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="120" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="120" rx="3" ry="3" width="100" height="24" />
         <rect x="0" y="160" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="160" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="160" rx="3" ry="3" width="100" height="24" />
         <rect x="0" y="200" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="200" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="200" rx="3" ry="3" width="100" height="24" />
         <rect x="0" y="240" rx="3" ry="3" width="150" height="24" />
-        <rect x="220" y="240" rx="3" ry="3" width="150" height="24" />
+        <rect x="220" y="240" rx="3" ry="3" width="100" height="24" />
       </ContentLoader>
```

Comment:
While working on this PR, I noticed that this loading animation was too wide for it's parent container.